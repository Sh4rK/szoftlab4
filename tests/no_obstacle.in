loadMap test.map
loadMission attack.mission
listTowers
step 15
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles
step 5
listEnemies
listProjectiles

exit
