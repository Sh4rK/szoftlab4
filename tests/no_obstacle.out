1 75.0 (3.6;3.6)
1 75.0 (4.9;4.9)
1 75.0 (6.2;6.2)
1 75.0 (7.5;7.5)
1 75.0 (8.8;8.8)
1 75.0 (10.1;10.1)
1 75.0 (11.4;11.4)
1 75.0 (12.7;12.7)
